etc/config/c++.amazon.properties

```diff
@@ -773,14 +773,44 @@ compiler.zapcc190308.name=x86-64 Zapcc 190308
 
 ###############################
 # Cross GCC
-group.cross.compilers=&ppcs:&mipss:&nanomips:&mrisc32:&msp:&gccarm:&avr:&rvgcc:&xtensaesp32:&xtensaesp32s2:&xtensaesp32s3:&platspec:&kalray:&s390x
+group.cross.compilers=&ppcs:&mipss:&nanomips:&mrisc32:&msp:&gccarm:&avr:&rvgcc:&xtensaesp32:&xtensaesp32s2:&xtensaesp32s3:&platspec:&kalray:&s390x:&sh
 group.cross.supportsBinary=true
 group.cross.groupName=Cross GCC
 group.cross.supportsExecute=false
 group.cross.licenseLink=https://gcc.gnu.org/onlinedocs/gcc/Copying.html
 group.cross.licenseName=GNU General Public License
 group.cross.licensePreamble=Copyright (c) 2007 Free Software Foundation, Inc. <a href="https://fsf.org/" target="_blank">https://fsf.org/</a>
 
+###############################
+# Cross for sh
+group.sh.compilers=&gccsh
+
+# GCC for sh
+group.gccsh.supportsBinary=true
+group.gccsh.supportsExecute=false
+group.gccsh.baseName=sh gcc
+group.gccsh.groupName=sh gcc
+group.gccsh.compilers=shg494:shg950:shg1220
+group.gccsh.isSemVer=true
+
+compiler.shg494.exe=/opt/compiler-explorer/sh/gcc-4.9.4/sh-unknown-elf/bin/sh-unknown-elf-g++
+compiler.shg494.semver=4.9.4
+compiler.shg494.objdumper=/opt/compiler-explorer/sh/gcc-4.9.4/sh-unknown-elf/bin/sh-unknown-elf-objdump
+compiler.shg494.demangler=/opt/compiler-explorer/sh/gcc-4.9.4/sh-unknown-elf/bin/sh-unknown-elf-c++filt
+compiler.shg494.name=sh 4.9.4
```

Comment:
`name` gets automatically set to `<baseName> <semver>` for `isSemVer=true` compilers, so if you set the `.baseName` to just `sh`, you can skip defining this `.name` for all three compilers :)